src/views/advanced-search/index.html

```diff
@@ -19,6 +19,7 @@
     <div class='govuk-form-group'>
       <div class='govuk-form-group' style='overflow: scroll; height: 200px; width: 623px; padding-left: 5px;'>
         <fieldset class='govuk-fieldset'>
+          <legend>
```

Comment:
Isn't the legend supposed to provide a brief description of what the fieldset contains so a screen reader can read it out to a blind or partially-sighted user, as described here?: https://www.tpgi.com/fieldsets-legends-and-screen-readers/#:~:text=The%20most%20important%20relationship%20is,the%20data%20or%20choices%20required.